test/e2e/integreatly_test.go

```diff
@@ -113,6 +112,16 @@ func integreatlyWorkshopTest(t *testing.T, f *framework.Framework, ctx *framewor
 			break
 		}
 	}
+	//SolutionExplorer Stage - verify operators deploy
+	products = map[string]string{
+		"solution-explorer": "tutorial-web-app-operator",
+	}
+	for product, deploymentName := range products {
+		err = waitForProductDeployment(t, f, ctx, product, deploymentName)
+		if err != nil {
+			break
+		}
+	}
```

Comment:
I don't like this change, since it still does not check whether the installation actually completed. I would prefer to create a PR to the e2e test and include your changes to the Makefile (there is also one additional step for solution explorer in the `make/e2e/olm`

Reply:
added your Makefile changes here: https://github.com/integr8ly/integreatly-operator/pull/153 on top of the e2e test changes. Do you wanna close this PR?

Reply:
The change here was fine since it was just adding the additional stage with the same level of testing as it always had, to allow the test to actually work. Since you have now rolled all these changes (I assume it has all the changes anyway, you wiped out my commit) into your PR https://github.com/integr8ly/integreatly-operator/pull/153, i guess we can close this one.